### PR TITLE
update to kryo 3.0.1 & akka 2.3.9

### PR DESCRIPTION
**Pull request by hepin1989**
update to support java8
https://github.com/twitter/chill/issues/223

### CODE REVIEW

**Comment by hepin1989** [comment]
test passed via sbt ++ 2.11.6,but not sbt ++ 2.11.2

File: chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala [hunk 1]
```diff
@@ -90,76 +87,76 @@ class KryoBase extends Kryo {
...
    * TODO this should be fixed in Kryo. When it is, remove this
    */
-  override def newInstantiator(cls: Class[_]) = {
-    import Instantiators._
-    newOrElse(cls,
-      List(reflectAsm _, normalJava _),
-      // Or fall back on the strategy:
-      tryStrategy(cls).newInstantiatorOf(cls))
-  }
+  //  override def newInstantiator(cls: Class[_]) = {
```
Comment (ianoc) [inline chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala]: Why are all these commented?

File: project/Build.scala [hunk 2]
```diff
@@ -152,7 +150,7 @@ object ChillBuild extends Build {
     resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/",
     libraryDependencies ++= Seq(
       "com.typesafe" % "config" % "1.2.1",
-      "com.typesafe.akka" %% "akka-actor" % "2.3.6" % "provided"
+      "com.typesafe.akka" %% "akka-actor" % "2.3.9" % "provided"
```
Comment (ianoc) [inline project/Build.scala]: Why the akka changes/why did the akka version change?

**Comment by ianoc** [comment]
Should do one thing at a time, the akka is a minor version bump which should be binary compatible, why are there any code changes/why make it at all? The 2.11 thing in travis you can bump the .travis.yml to a newer version of 2.11 to see if it passes there

**Comment by hepin1989** [comment]
i will re do this tomorrow😋

File: chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala (hunk 1 above)
Comment (hepin1989) [inline chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala]: this workarould should be fixed.

**Comment by hepin1989** [comment]
need a fix in kryo ,reopen later

**Pull request closed without merging**